pkl-executor/pkl-executor.gradle.kts

```diff
@@ -38,6 +38,10 @@ dependencies {
 
   testImplementation(projects.pklCommonsTest)
   testImplementation(projects.pklCore)
+  testImplementation(libs.truffleApi)
+  testImplementation(libs.graalSdk)
+  testImplementation(libs.svm)
+  testImplementation(libs.truffleSvm)
```

Comment:
Why are these libraries added as test dependencies? They should be bundled in the fat jar already; the test shouldn't need to depend on them.

Same question goes for pkl-cli: why add these libraries as implementation dependencies?

Reply:
I don't know why they were required before and aren't now, but removing them didn't break anything. Fixed